pl_examples/test_examples.py

```diff
@@ -25,6 +28,29 @@ def test_gpu_template(cli_args):
         run_cli()
 
 
+@pytest.mark.parametrize('cli_args', ['--epochs 1 --gpus 1'])
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+def test_imagenet(tmpdir, cli_args):
```

Comment:
nice! so it will just run 5 images, right?

Reply:
Wait. there may be a problem related to drop_last here.

Reply:
@Borda I can't pass python -m pytest pl_examples/test_examples.py anymore.

Reply:
then pass in batch_size=5 as arg in the test and you should be fine.

Reply:
doing it.

Reply:
Never mind. drop_last is False. There will be only one batch for train and on batch for val. BTW it's 20 images in total.